src/commands/cloud-run/flare.ts

```diff
@@ -114,21 +130,60 @@ export class CloudRunFlareCommand extends Command {
     const configStr = util.inspect(config, false, 10, true)
     this.context.stdout.write(`\n${configStr}\n`)
 
-    // Save and zip service configuration
-    this.context.stdout.write(chalk.bold('\n💾 Saving configuration...\n'))
-    const rootFolderPath = path.join(process.cwd(), FLARE_OUTPUT_DIRECTORY)
+    // Get logs
+    const logFileMappings = new Map<CloudRunLog[], string>()
+    if (this.withLogs) {
+      this.context.stdout.write(chalk.bold('\n📖 Getting logs...\n'))
+
+      const logsConfig: LogConfig[] = [
+        {type: 'total', fileName: ALL_LOGS_FILE_NAME},
+        {type: 'warning', severityFilter: ' AND severity>="WARNING"', fileName: WARNING_LOGS_FILE_NAME},
+        {type: 'error', severityFilter: ' AND severity>="ERROR"', fileName: ERRORS_LOGS_FILE_NAME},
+        {type: 'debug', severityFilter: ' AND severity="DEBUG"', fileName: DEBUG_LOGS_FILE_NAME},
+      ]
```

Comment:
This seems to be a constant?

Reply:
True, this can be moved to the top of the file.